Utilities/CMakeLists.txt

```diff
@@ -8,4 +8,4 @@
   see <http://creativecommons.org/licenses/by-sa/4.0/>.
 ]]
 
-# TODO(Gordon) Create target for Utilities
```

Comment:
This is still a TODO, we haven't added a target for the utilities (if we're deciding not to, we can just remove the file).

Reply:
Ok, leaving it untouched. Done.